panther_analysis_tool/main.py

```diff
@@ -322,7 +322,7 @@ def setup_global_helpers(global_analysis: List[Any]) -> List[Any]:
         # If the module could not be loaded, continue to the next
         if load_err:
             invalid_specs.append((analysis_spec_filename, load_err))
-            break
+            continue
```

Comment:
Should we be logging a message here?

Reply:
We add the filename and err to the `invalid_specs` dict that gets passed around and eventually printed at the end so it should be logged eventually already.